USING_ELEKTRA.md

```diff
@@ -0,0 +1,252 @@
+# Using Elektra
+
+In the new version of LCDproc, [Elektra](https://www.libelektra.org/home) 
+will be used to store and edit configuration settings which are used by LCDproc.
+
+Elektra stores configuration in a global, hierarchical key database. 
+Elektra furthermore provides the possibility to have a 
+[validation on configuration settings](https://www.libelektra.org/tutorials/validate-configuration)
+because of configuration specifications. A configuration specification 
+simply tells which values should be legal for a certain configuration setting,
+eg. the server `port` value should be an integer within 0 and 65535.
+
+This tutorial will teach you how change configuration and introspect the specification. 
+Be sure to have everything installed correctly by first 
+ through [INSTALL.md](INSTALL.md).
+
+## Basic Concept
+
+Elektra stores values in form of a key value pair in a global, 
+hierarchical key database. They are saved below a *path* where a path
+is similar to linux directories. If you have followed the specification installation process correctly
+from the [INSTALL.md](INSTALL.md), you now have the full specification 
+[mounted](https://www.libelektra.org/tutorials/mount-configuration-files)
+under the `spec` [namespace](https://www.libelektra.org/tutorials/namespaces).
+Namespaces are used for [cascading lookups](https://www.libelektra.org/tutorials/cascading)
+so the application knows where to search for configuration settings. In case of LCDproc,
+the `spec` namespace is first looked at since it contains metadata and afterwards the `user`
```

Comment:
Usually, "system" will be the namespace, as LCDproc is usually configured sytem-wide. "user" and "dir" might be possible to. Users might ask how they can disable "user" and "dir" config. (via the "namespace" meta data)

Reply:
Where do you need to set `namespace/#0 = system`? On every key or just on the root key?

Reply:
At every single key where you want it. We would need a plugin that copies that metadata everywhere. I think for now we can simply ignore this feature, as this is only nice-to-have.

Reply:
If it is a nice-to-have and you need to add it to all namespaces I think it just confuses users. It should not be in the tutorial yet